BonusCalcApi/V1/UseCase/UpdateReportsSentAtUseCase.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Threading.Tasks;
+using BonusCalcApi.V1.Gateways.Interfaces;
+using BonusCalcApi.V1.Infrastructure;
+using BonusCalcApi.V1.UseCase.Interfaces;
+
+namespace BonusCalcApi.V1.UseCase
+{
+    public class UpdateReportsSentAtUseCase : IUpdateReportsSentAtUseCase
```

Comment:
UpdateReportSentAtUseCase and UpdateReportsSentAtUseCase are very similarly named and we'll get confusion in the future in this instance. Can we please change one so it's clearer. This usecase is more about updating weeks - perhaps UpdateWeeksReportUseCase or something like that